@here/harp-mapview/lib/MapView.ts

```diff
@@ -546,6 +552,22 @@ export interface MapViewOptions {
      */
     backgroundTilingScheme?: TilingScheme;
 
+    /**
+     * Should be the [[PolarTileDataSource]] used on spherical projection.
```

Comment:
What is the default?

Reply:
there is `const DEFAULT_POLAR_STYLE_SET_NAME = "polar";` above

Reply:
think, I should left a comment here

Reply:
added comment